Assets/cart.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Instead of SVG images, you can use icons.

Reply:
i have used icon...forget to remove images from file